core/server/services/members/api.js

```diff
@@ -69,7 +70,11 @@ const issuer = siteOrigin;
 const ssoOrigin = siteOrigin;
 let mailer;
 
-const membersConfig = config.get('members');
+const membersConfig = settingsCache.get('members_subscription_settings');
```

Comment:
Has the `contentApiAccess` been added to the member settings? Because this will break the `validateAudience` method if not - possibly wanna call it `membersSettings` so we don't conflict?

Reply:
Ah good catch, missed the `contentApiAccess` being used in the file. I'll rename the settings to avoid conflict. 👍